feature/qos/tests/qos_ecn_config_test/README.md

```diff
@@ -8,45 +8,59 @@ Verify QoS ECN feature configuration.
 
 *   Connect DUT port-1 to ATE port-1, DUT port-2 to ATE port-2.
 
-*   ECN config:
+*   ECN configuration paramentes:
+
+    *   This test is verifying DUT ability to accept ECN configuration with vertical buffer utilization cut-off line.\ If buffer is utilized below that cut-off value no packet is ECN CE marked.\ If buffer is utilized at of above that cut-off value all packet are ECN CE marked.
 
     *   ECN profile can be created for different queues. ECN profiles per queue
         can be applied to the output side of interfaces.
 
-        min-threshold | max-threshold | enable-ecn | drop  | weight  | max-drop-probability-percent
-        ------------- | ------------- | ---------- | ----- | ------- | ----------------------------
-        80000         | 2^64-1        | true       | false | not set | 1
-
-        *   Note: max-threshold is set to max uint64 value 2^64-1
-            or 18446744073709551615.
-
-    *   Validate that the following values can be configured
-
-        *   min-threshold
-        *   max-threshold
-        *   enable-ecn
-        *   drop
-        *   max-drop-probability-percent
-
-    *   The following OC config paths can be used to configure the above values:
-
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/min-threshold
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/max-threshold
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/enable-ecn
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/weight
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/drop
-        *   qos/queue-management-profiles/queue-management-profile/wred/uniform/config/max-drop-probability-percent
-
-*   Interfaces
-
-    *   Validate ECN profile can be applied under output interface queue using
-        OC config path:
-        *   /qos/interfaces/interface/output/queues/queue/config/queue-management-profile
+        Test case|min-threshold | max-threshold | enable-ecn | drop  | weight  | max-drop-probability-percent
+        |--------|------------- | ------------- | ---------- | ----- | ------- | ----------------------------
+        |#1      |80KB          |80KB*          | true       | false | not set | 100
+        |[TODO]#2      |3.125MB       | 6.250MB       | true       | false | not set | 100
+        |[TODO]#3      |1%            | 2%            | true       | false | not set | 100
+
+    * 6.25MB max-treshhold is selected as it represents ~500 mico-seconds of Delay bandwidth Buffer on 100GE interfaces. What is O(2%) of buffer depth, hence allows for micro-burst absorbtion without beackpressing senders and at same time leaves enough DBB for accomodate RTT ECN signaling loop delay in global network for longer burst/congestion.
+
+*   Procedure
+    * Test Case #1 80KB min-threshold equal max-threshold
```

Comment:
Should one of the line item be
- Ensure that at buffer utilization below the min-threshold, packets are not ECN CE marked
- Ensure that at buffer utilization between min and max thresholds, 100% of the packets are ECN marked
- Ensure that at buffer utilization above max-threshold, 100% of the packets are ECN marked

Reply:
That woud be separate test that is HW specific. 
This one is just update to existing test that can be run on virtual device / KNE